### PR TITLE
Add switch platform to honeywell

### PR DESCRIPTION
**Pull request by mkmer**
## Proposed change

Add the new Emergency heat switch and switch platform to the docs after removing aux heat from climate entity. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/114110
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/honeywell.markdown [hunk 1]
```diff
@@ -61,3 +61,11 @@ This integration will add Home Assistant sensors for the following:
 |Outdoor humidity | Average humidity of all Honeywell Wireless Outdoor Sensors|
 |Indoor temperature | Current temperature as measured at the specific thermostat|
 |Indoor humidity | Current humidity as measured at the specific thermostat|
+
+## Switch
+The switch platform integrates the emergency heat option as a switch for each device.  If the thermostat supports emergency heat, the switch entity will be created.
+
+This integration will add Home Assistant switch for the following:
```
Comment (c0ffeeca7) [inline source/_integrations/honeywell.markdown]:
```suggestion
This integration will add a switch for the following:
```

File: source/_integrations/honeywell.markdown [hunk 2]
```diff
@@ -61,3 +61,11 @@ This integration will add Home Assistant sensors for the following:
 |Outdoor humidity | Average humidity of all Honeywell Wireless Outdoor Sensors|
 |Indoor temperature | Current temperature as measured at the specific thermostat|
 |Indoor humidity | Current humidity as measured at the specific thermostat|
+
+## Switch
```
Comment (c0ffeeca7) [inline source/_integrations/honeywell.markdown:67]:
```suggestion
## Switch

```

File: source/_integrations/honeywell.markdown [hunk 3]
```diff
@@ -61,3 +61,11 @@ This integration will add Home Assistant sensors for the following:
...
+The switch platform integrates the emergency heat option as a switch for each device.  If the thermostat supports emergency heat, the switch entity will be created.
```
Comment (c0ffeeca7) [inline source/_integrations/honeywell.markdown]:
```suggestion
The switch entity integrates the emergency heat option for each device.  If the thermostat supports emergency heat, the switch entity will be created.
```

File: source/_integrations/honeywell.markdown [hunk 4]
```diff
@@ -61,3 +61,11 @@ This integration will add Home Assistant sensors for the following:
...
+
+This integration will add Home Assistant switch for the following:
+|Sensor|Value|
```
Comment (c0ffeeca7) [inline source/_integrations/honeywell.markdown]:
```suggestion
|Switch|Value|
```

File: source/_integrations/honeywell.markdown [hunk 5]
```diff
@@ -25,8 +26,7 @@ If your system is compatible with this integration, then you will be able access
 - [Supported hardware](#supported-hardware)
 - [Climate](#climate)
 - [Sensor](#sensor)
-
-{% include integrations/config_flow.md %}
+- [Switch](#switch)
```
Comment (c0ffeeca7) [inline source/_integrations/honeywell.markdown:29]:
```suggestion
- [Switch](#switch)

{% include integrations/config_flow.md %}
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @mkmer :+1:

File: source/_integrations/honeywell.markdown (hunk 5 above)
Comment (mkmer) [reply source/_integrations/honeywell.markdown:29]: Thanks!

**Pull request merged**